### PR TITLE
WRAPR-95 more feedback

### PR DESCRIPTION
**Pull request by mogmarsh**
fix pro story layout

### CODE REVIEW

**Review by dlh01 — APPROVED** [review]
🌳

File: layouts/pro-multi-story.html [hunk 1]
```diff
@@ -2,26 +2,26 @@
...
-<!-- wp:newsletter-builder/post {"showCta":false,"order":["image","title","byline","excerpt","content","cta"]} /-->
+<!-- wp:newsletter-builder/post {"showContent":true,"showExcerpt":false,"showCta":true,"order":["image","title","byline","excerpt","content","cta"]} /-->
 
 <!-- wp:separator -->
 <hr class="wp-block-separator has-alpha-channel-opacity"/>
 <!-- /wp:separator -->
 
 <!-- wp:newsletter-builder/two-up-post -->
-<div class="wp-block-newsletter-builder-two-up-post"><!-- wp:newsletter-builder/post {"showCta":false,"showExcerpt":false,"showContent":false,"showByline":false,"order":["image","title","excerpt","content","cta","byline"]} /-->
+<div class="wp-block-newsletter-builder-two-up-post"><!-- wp:newsletter-builder/post {"showContent":true,"showExcerpt":false,"showCta":false,"showExcerpt":false,"showContent":false,"showByline":false,"order":["image","title","excerpt","content","cta","byline"]} /-->
```
Comment (kaitlinbolling) [inline layouts/pro-multi-story.html:16]: It looks like we're defining `showContent` and `showExcerpt` twice here. I think they should both be false?

Comment (mogmarsh) [reply layouts/pro-multi-story.html:16]: @kaitlinbolling thanks! will fix

**Review by kaitlinbolling — APPROVED** [review]
😎

**Pull request merged**